src/playermat/Playmat.ttslua

```diff
@@ -680,7 +622,7 @@ function showDrawButton(visible)
   -- remove the "Draw 1" button
   else
     local buttons = self.getButtons()
-    for i = 1, #buttons do 
+    for i = 1, #buttons do
```

Comment:
Is this a LF related change?

Reply:
I'm not sure honestly - doesn't look like it, most likely another trailing space I stumbled across and cleaned up.